html/janus.nojquery.js

```diff
@@ -22,248 +22,29 @@
 	OTHER DEALINGS IN THE SOFTWARE.
  */
 
-// List of sessions
-Janus.sessions = {};
-
-// Screensharing Chrome Extension ID
-Janus.extensionId = "hapfgfdkleiggjjpfpenajgdnfckjpaj";
-Janus.isExtensionEnabled = function() {
-	if(window.navigator.userAgent.match('Chrome')) {
-		var chromever = parseInt(window.navigator.userAgent.match(/Chrome\/(.*) /)[1], 10);
-		var maxver = 33;
-		if(window.navigator.userAgent.match('Linux'))
-			maxver = 35;	// "known" crash in chrome 34 and 35 on linux
-		if(chromever >= 26 && chromever <= maxver) {
-			// Older versions of Chrome don't support this extension-based approach, so lie
-			return true;
-		}
-		return (document.getElementById('janus-extension-installed') !== null);
+/**
+ * UMD wrapper.
+ */
+(function (root, factory) {
+	if (typeof define === "function" && define.amd) {
+		define([
+			"adapter",
+		], function(adapter) {
+			return (root.Janus = factory(adapter));
+		});
```

Comment:
@lminiero this is the exact point of the UMD wrapper -- those sections you're referencing above handle the case when a user wants to load the library via RequireJS. The next case supports if a user wants to load the library via Node.js/CommonJS, and the final case if the user wants to load the library in the old school global scope (which is what the demos do). Removing those lines would defeat the purpose of the wrapper :)